packages/gatsby-theme-docs/src/components/theme-provider.js

```diff
@@ -29,9 +36,16 @@ const ThemeProvider = (props) => {
   return (
     <ErrorBoundary>
       <SiteDataContext.Provider value={siteData}>
-        <Reset />
-        <Globals />
-        {props.children}
+        <CodeExampleLanguageContext.Provider
+          value={{
+            codeExampleLanguage,
+            setCodeExampleLanguage,
+          }}
+        >
```

Comment:
FYI: now that the code-examples are an add-on, we need to find another solution for this. I would suggest maybe to re-open a new PR with a fresh approach.